### ISSUE TITLE
Bevy v0.15

### ISSUE THREAD

**Issue by SMITTY-94** [kulkalkul/bevy_mod_billboard#30]
Hello there! 

Are there any plans to upgrade to Bevy 0.15?

**Comment by kulkalkul** [comment]
Yes, but I don't have any spare time atm; so can't say any predictions on when I'll do it. Though it is highly unlikely to happen before new year 😕

**Comment by The-DevBlog** [comment]
No worries. Thanks for the response. Merry Christmas!

**Comment by voximity** [comment]
Hey, I've been working on migrating this to 0.15 - I have billboarded textures working and am working on billboarded text now. Hoping to have a PR made in the next day or so.
